hystrix/src/main/java/feign/hystrix/HystrixInvocationHandler.java

```diff
@@ -110,7 +113,31 @@ protected Object getFallback() {
           return super.getFallback();
         }
         try {
-          Object result = fallbackMethodMap.get(method).invoke(fallback, args);
+          Method fallbackMethod = fallbackMethodMap.get(method);
+          Method enhancedFallbackMethod = null;
```

Comment:
the word enhanced doesn't help explain anything. how about something like `fallbackMethodWithThrowable`?